homework/styles.css

```diff
@@ -0,0 +1,165 @@
+body {
+  background-color: #111827;
+  color: white;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+.shows {
+  max-width: 700px;
+}
+
+.show-card {
+  display: flex;
+  flex-direction: column;
+  background-color: #1f2937;
+  border-radius: 20px;
+  font-size: larger;
```

Comment:
Ako slika već mora zauzimat 130% parenta, onda bi ovdje trebali dodat nešto ovako:
`overflow: hidden;`